src/bootstrap_gui.cpp

```diff
@@ -278,6 +278,69 @@ class BootstrapAskForDownloadWindow : public Window, ContentCallback {
 
 #endif /* defined(WITH_FREETYPE) */
 
+#if defined(__EMSCRIPTEN__)
+#include <emscripten.h>
+#include "network/network.h"
+#include "network/network_content.h"
+#include "openttd.h"
+#include "video/video_driver.hpp"
+
+class BootstrapEmscripten : public ContentCallback {
+	bool downloading = false;
+	uint total_files;
+	uint total_bytes;
+	uint downloaded_bytes;
```

Comment:
please initialize these values. At least ,how I read it, `downloaded_bytes` is never actually set to zero, but printed nevertheless :)

total_files and total_bytes strictly seen are initialized before use, but it might just be good to assign values to them, to make perfectly clear they are always initialized correctly.